GVFS/GVFS.Common/InstallerPreRunChecker.cs

```diff
@@ -188,9 +188,10 @@ private bool IsGVFSUpgradeAllowed(out string consoleError)
 
             if (!this.IsGVFSUpgradeSupported())
             {
+                string prjfsError = "is only supported after the \"Windows Projected File System\" optional feature has been enabled by a manual upgrade of VFS for Git, and only on versions of Windows that support this feature.";

```

Comment:
manual upgrade? -> manual installation?

Reply:
also - could string literal be inlined directly below (as it was before)? doesn't really matter, but if we are using it directly without modification, I think it is easier to read inline.